### PR TITLE
Some small doc improvements

### PR DESCRIPTION
**Pull request by ddeboer**

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -110,10 +110,11 @@
 
 # The name for this set of Sphinx documents.  If None, it defaults to
 # "<project> v<release> documentation".
-#html_title = None
+# Don't show version in title
+html_title = "FOSHttpCache Documentation"
```
Comment (ddeboer) [inline doc/conf.py]: This hides the version from the title when viewing the docs. Currently foshttpcache.readthedocs.org is listed as ‘FOSHttpCache 1.0.0 documentation’ which may be somewhat confusing as we’re already at 1.3.0.

File: doc/conf.py [hunk 2]
```diff
@@ -110,10 +110,11 @@
...
 
 # A shorter title for the navigation bar.  Default is the same as html_title.
-#html_short_title = None
+html_short_title = FOSHttpCache
```
Comment (dbu) [inline doc/conf.py]: seems the example was misleading. this needs quotes i think. travis doc building fails.

File: doc/conf.py (hunk 1 above)
Comment (dbu) [inline doc/conf.py]: agreed. we could make it "FOSHttpCache Documentation (current)" though,
to make it more obvious, if you want.

**Comment by ddeboer** [comment]
Added the quotes, this should build now.

File: doc/conf.py (hunk 1 above)
Comment (ddeboer) [inline doc/conf.py]: As we’re only maintaining one major version now, I left that out for the time being. When we start having docs for version 2 as well as 1, we can always add it in.

**Comment by dbu** [comment]
:+1:

**Pull request merged**